### PR TITLE
Validate Points in the EE.

### PR DESCRIPTION
**Pull request by BillWhite**
The EE function pointfromtext should validate longitude and latitude.

https://issues.voltdb.com/browse/ENG-9746

### CODE REVIEW

File: tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java [hunk 1]
```diff
@@ -747,15 +747,30 @@ private void assertGeographyValueWktParseError(Client client, String expectedMsg
...
+        // Try a couple of bad latitudes.
+        assertSelectGeographyPointValueWktParseError(client, "Latitude must be in the range", "point(10 100)");
+        assertInsertGeographyPointValueWktParseError(client, "Latitude must be in the range", "point(10 100)", 100);
+        assertSelectGeographyPointValueWktParseError(client, "Latitude must be in the range", "point(20 -100)");
+        assertInsertGeographyPointValueWktParseError(client, "Latitude must be in the range", "point(20 -100)", 101);
+        // Try a couple of bad longitudes.
+        assertSelectGeographyPointValueWktParseError(client, "Longitude must be in the range", "point(200 20)");
+        assertInsertGeographyPointValueWktParseError(client, "Longitude must be in the range", "point(200 20)", 200);
+        assertSelectGeographyPointValueWktParseError(client, "Longitude must be in the range", "point(-200 20)");
+        assertInsertGeographyPointValueWktParseError(client, "Longitude must be in the range", "point(-200 20)", 201);
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java]: I would also check boundary values here too... 90.0, -90.0 should be valid latitudes and 180.0, -180.0 should be valid longitudes.

**Comment by wolffcm** [comment]
Looks good other than my one comment.

File: tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java (hunk 1 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java]: There is a test called testPointFromTextPositive above which tests this.  I don't remember if it was there when you made this comment, but it's there now.

File: tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java [hunk 2]
```diff
@@ -597,7 +597,7 @@ public void testGroupBy() throws Exception {
         }
     }
 
-    public void testUpdate() throws Exception {
+    public void notestUpdate() throws Exception {
```
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestGeographyValueQueries.java]: All these notest tests will be changed to test tests.  I forgot to do that.
